docs/src/main/asciidoc/security-built-in-authentication.adoc

```diff
@@ -7,17 +7,46 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 
 include::./attributes.adoc[]
 
-This document describes the Quarkus built-in authentication mechanisms for HTTP based FORM, BASIC and Mutual TLS authentication as well as the proactive authentication.
+The following section describes the Quarkus built-in authentication mechanisms for HTTP based FORM, BASIC, and Mutual TLS authentication. Proactive authentication is also described.
 
 [[basic-auth]]
 == Basic Authentication
+ 
+HTTP Basic authentication uses fields in the HTTP header, and is the easiest to set up and least resource demanding technique for enforcing access controls to Web resources without requiring HTTP cookies, session identifiers, or login pages.
+
+In the context of an HTTP request, basic access authentication is a method for an HTTP user agent (such as a Web browser) to provide a user name and password when making a request. In basic HTTP authentication, a request contains a header field in the form of _Authorization: Basic_ _<credentials>_, where credentials are the Base64 encoding of a user ID and password joined by a colon. 
```

Comment:
```suggestion
In the context of an HTTP request, Basic authentication is a method for an HTTP user agent (such as a Web browser) to provide a user name and password when making a request. A request contains a header field in the form of _Authorization: Basic_ _<credentials>_, where credentials are the Base64 encoding of a user ID and password joined by a colon. 
```

Reply:
`Basic authentication` is used everywhere so I propose to use the same phrase. Also, if you don't mind, please change `Basic authentication` to `Basic Authentication`, it seems like the latter option is more frequently used (even in the PR title :-) ) - but I'll leave it up to you